arcgis-ios-sdk-samples/Maps/Show location history/README.md

```diff
@@ -0,0 +1,14 @@
+# Show location history
```

Comment:
Did you write this README yourself? There is a README template for this sample as part of the sample design. If someone didn't tell you about that, I'm truly sorry. I'll send you the link. Please use the template as the basis of the README, tweaking it as needed to make it specific to iOS. Like adding the `AGS` class prefix, for example.

Reply:
I was directed to the `README` you shared, but there were fewer similarities than I anticipated. Moreover, I saw a fair bit of inconsistency across other samples as far as the content and structure of each file.

In any case, [`e8d8235`](https://github.com/Esri/arcgis-runtime-samples-ios/pull/751/commits/e8d82357ea8675704803bc542acafc6859f8d3e9) should address the concerns you noted.